pdr_backend/statutil/test/test_dist.py

```diff
@@ -0,0 +1,63 @@
+from typing import List
+
+from enforce_typing import enforce_types
+import numpy as np
+from plotly.subplots import make_subplots
+
+from pdr_backend.statutil.dist import (
+    add_pdf,
+    plot_pdf,
+    plot_pdf_cdf_nq,
+    plot_pdf_nq,
+)
+
+SHOW_PLOT = True # only turn on for manual testing
```

Comment:
could we retrieve this from the local user env? That way, you can export it on individual workspaces without always having the tests fail.

Reply:
This is a good idea. I'll do this here and create an issue for the other places we have that pattern.

Reply:
https://github.com/oceanprotocol/pdr-backend/issues/1211